packages/react/use-funnel/src/useFunnel.tsx

```diff
@@ -56,13 +56,16 @@ export const useFunnel = <Steps extends NonEmptyArray<string>>(
     () =>
       Object.assign(
         function RouteFunnel(props: RouteFunnelProps<Steps>) {
-          // eslint-disable-next-line react-hooks/rules-of-hooks
-          const step = useQueryParam<Steps[number]>(stepQueryKey) ?? options?.initialStep;
+          const step = useQueryParam(stepQueryKey) ?? options?.initialStep;
```

Comment:
removed unnecessary type assertion in useFunnel

Reply:
This is unnecessary change in this PR.